tests/framework-tests/preferences/js/SyllabificationEnactorTests.js

```diff
@@ -219,8 +244,8 @@ https://github.com/fluid-project/infusion/raw/master/Infusion-LICENSE.txt
                 ]
             },
             markup: {
-                injectWhenDisabled: "<p class=\"flc-syllabification-injectWhenDisabled\">Global temperature has increased over the past 50 years.</p>",
-                injectWhenEnabled: "<p class=\"flc-syllabification-injectWhenEnabled\">Global temperature has increased over the past 50 years.</p>"
+                injectWhenDisabled: "<p class=\"flc-syllabification-injectWhenDisabled\">Global temperature has in&shy;creased over the past 50 years.</p>",
+                injectWhenEnabled: "<p class=\"flc-syllabification-injectWhenEnabled\">Global temperature has in&shy;creased over the past 50 years.</p>"
```

Comment:
Could be helpful to add a comment here to note that one purpose of the odd-looking text is to check FLUID-6554